code/datums/gamemode/misc_gamemode_procs.dm

```diff
@@ -397,6 +397,44 @@
 	killer.laws.zeroth_lock = TRUE
 	to_chat(killer, "New law: 0. [law]")
 
+/proc/equip_time_agent(var/mob/living/carbon/human/H, var/datum/role/time_agent/T)
```

Comment:
This may need to force-unfat people, otherwise they could spawn without the timesuit. (it would get deleted)